README.md

```diff
@@ -1,68 +1,57 @@
-# ibctest
+<div align="center">
+<h1>ibctest</h1>
 
 [![Go Reference](https://pkg.go.dev/badge/github.com/strangelove-ventures/ibctest@main.svg)](https://pkg.go.dev/github.com/strangelove-ventures/ibctest@main)
+[![License: Apache-2.0](https://img.shields.io/github/license/strangelove-ventures/ibctest.svg?style=flat-square)](https://github.com/strangelove-ventures/ibctest/blob/main/create-test-readme/LICENSE)
+[![Go Report Card](https://goreportcard.com/badge/github.com/strangelove-ventures/ibctest)](https://goreportcard.com/report/github.com/strangelove-ventures/ibctest)
 
-ibctest orchestrates Go tests that utilize Docker containers for multiple
+
+
+`ibctest` orchestrates Go tests that utilize Docker containers for multiple
 [IBC](https://docs.cosmos.network/master/ibc/overview.html)-compatible blockchains.
 
-## Focusing on Specific Tests
+It allows users to quickly spin up custom testnets and dev environments to test IBC and chain infrastructures.
+</div>
 
-You may focus on a specific tests using the `-test.run=<regex>` flag.
+### -- Features --
 
-```shell
-ibctest -test.run=/<test category>/<chain combination>/<relayer>/<test subcategory>/<test name>
-```
+- Built-in suite of conformance tests to test high-level IBC compatibility between chain sets
+- Easily construct customized tests in highly configurable environments
+- Deployable as CI tests in production workflows
 
-If you want to focus on a specific test:
+<br>
 
-```shell
-ibctest -test.run=/////relay_packet
-ibctest -test.run=/////no_timeout
-ibctest -test.run=/////height_timeout
-ibctest -test.run=/////timestamp_timeout
-```
+## Table Of Contents
+- [Building Binary](#building-binary)
+- **Usage:**
+    - [Running Conformance Tests](./docs/conformanceTests.md) - suite of built-in tests that test high-level IBC compatibility
+    - [Architect Custom Tests](./docs/architectCustomTests.md) - How to create custom tests
+- [Retaining Data on Failed Tests](./docs/retainingDataOnFailedTests.md)
+- [Deploy as GitHub CI Tests](./docs/ciTests.md)
```

Comment:
I ❤️ this pattern. The primary README is an index to smaller, focused documentation.